proposals/2022-04-min-commission-rate/README.md

```diff
@@ -0,0 +1,47 @@
+# Signaling Proposal: Upgrade the chain for introducing a min-commission-rate parameter and setting it to 3%
+
+
+## Proposal
+
+Previously, the proposal #6 "Increase minimum commission rate to 5%" was submitted but was rejected.<br>
+https://www.mintscan.io/medibloc/proposals/6
+
+This topic has been discussed for a very long time in the Cosmos ecosystem.<br>
+https://forum.cosmos.network/t/proposal-are-validators-charging-0-commission-harmful-to-the-success-of-the-cosmos-hub/2505
+
+Although proposal #6 was reasonable enough, MediBloc team also rejected it because we didn't have clear strategies and implementations for enforcing validators to increase their commission rate.
+
+Also, the team thought that the ideal network should work without a minimum commission rate. So, instead of having the restriction, we have convinced some validators to set a reasonable commission rate. Thankfully, many validators have increased their commission rate. But, that is the inefficient and not-decentralized way. Instead, we have found that it is beneficial to have a minimum commission rate at least while the chain is in early-stage. This will improve network sustainability and make it possible for validators to compete in healthy ways without setting the commission rate to 0%.
```

Comment:
```suggestion
Also, the team thought that the ideal network should work without a restriction on commission rate. So, instead of implementing the restriction, the team has convinced those validators with 0% commision rate to raise its commission rate to a reasonable rate. Thankfully, many validators have cooperated with us and raised their commision rates. But, after doing so, we realized that our approach is the inefficient and non-decentralized way. Hence, we have found it beneficial to have a minimum commission rate at least while the chain is in the early-stage. We belive this will improve the network sustainability and make it possible for validators to compete in healthy ways without setting the commission rate to 0%.
```